src/vshaxe/commands/Commands.hx

```diff
@@ -72,6 +77,47 @@ class Commands {
 		});
 	}
 
+	function extendedTyping(args) {
+		if (args.text == "{")
+			indentCurlyBracket();
+		commands.executeCommand('default:type', args);
+	}
+
+	// With possible comments after bracket
+	final lineEndsWithOpenBracket = new RegExp("[([{]\\s*(\\/\\/.*|\\/[*].*[*]\\/\\s*)?$");
```

Comment:
Seems more natural to use a Haxe regex here, can avoid escaping that way with a regex literal.